chain/neutrino.go

```diff
@@ -101,6 +101,8 @@ func (s *NeutrinoClient) Stop() {
 	s.started = false
 	s.clientMtx.Unlock()
 
+	s.wg.Wait()
```

Comment:
This now conflicts with the role of the `WaitForShutdown `  method.

Reply:
Reverted.